mlflow/store/tracking/file_store.py

```diff
@@ -680,3 +680,6 @@ def log_batch(self, run_id, metrics, params, tags):
                 self.set_tag(run_id, tag)
         except Exception as e:
             raise MlflowException(e, INTERNAL_ERROR)
+
+    def update_artifacts_location(self, run_id, new_artifacts_location):
+        raise ValueError("Use FileStore, aritfacts cannot be moved")
```

Comment:
I would change for something like 'Artifacats cannot be moved when using FileStore backend.'

Reply:
Done